<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scene1</class>
 <widget class="QMainWindow" name="scene1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>scene1</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button1.png);
border-image: url(:/button1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button1.png);
border-image: url(:/button1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/heal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/exit.png);
border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>head</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>body</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>legs</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>581</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 italic 11pt &quot;Arial Black&quot;;
border-image: url(:/textboard.png);
border-image: url(:/textboard.png);
text-decoration: underline;</string>
    </property>
   </widget>
   <zorder>button1</zorder>
   <zorder>button2</zorder>
   <zorder>button3</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>progressBar</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>textBrowser</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
